sakip_sabanci_config.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'traject_plus'
+require 'dlme_json_resource_writer'
+require 'macros/dlme'
+require 'macros/bnf'
+
+extend Macros::DLME
+extend TrajectPlus::Macros
+extend TrajectPlus::Macros::Xml
+
+NS = { xmlns: 'http://www.openarchives.org/OAI/2.0/' }.merge(Macros::BNF::NS).freeze
+
+settings do
+  provide 'writer_class_name', 'DlmeJsonResourceWriter'
+  provide 'reader_class_name', 'TrajectPlus::XmlReader'
+end
+
+record = '/xmlns:record/xmlns:metadata/oai_dc:dc'
+
+# Cho Required
+to_field 'id', extract: extract_xml("#{record}/dc:identifier", NS), transform: transform(strip: true)
```

Comment:
instead of `extract_xml("#{record}/dc:identifier", NS)` , consider `include Macros::OAI` and using `extract_oai('dc:identifier')`